words/objects.txt

```diff
@@ -1351,39 +1754,63 @@ mousepad
 moustache
 mouth
 move
+mozzarella
+muenster
+mulberry
 mule
+mum
+munchkin
+muscari
 muscle
 muse
 museum
 mushroom
 music
 musician
+muskmelon
 muskox
 mustang
 mustard
+myrtle
 myth
 nail
 name
+nance
```

Comment:
I would pull this one, it's a tree species but it can also be short hand for nancy (a girls name often used with hate against boys seen as "too girly")